global-files/.github/workflows/sync-labels.yml

```diff
@@ -7,7 +7,7 @@ on:
       - .github/labels.yml
       - .github/workflows/sync-labels.yml
   schedule:
-    - cron: 6 6 6 * *
+    - cron: 0 0 1 * *
```

Comment:
I still think it's bad luck to not use a number that doesn't mean "everything goes smoothly" here.

Reply:
Well, I think it's bad luck to use the number of the beast here 🤷

Reply:
That's just extra luck! This script is a beast, so it fits even better.

Reply:
Lol